src/redisgears.h

```diff
@@ -5,8 +5,15 @@
  *      Author: meir
  */
 
-#ifndef SRC_REDISGEARG_H_
-#define SRC_REDISGEARG_H_
+#pragma once
+
+#ifdef GEARS_API_MAIN
+#define GEARS_API
+#endif
+
+#ifndef GEARS_API
+#define GEARS_API extern
+#endif
```

Comment:
If we are already doing this change then why not also add a suffix like the redismodule.h?